spring-boot-tests/spring-boot-smoke-tests/spring-boot-smoke-test-data-redis/src/main/java/smoketest/data/redis/SampleRedisApplication.java

```diff
@@ -40,8 +41,10 @@ public void run(String... args) throws Exception {
 	}
 
 	public static void main(String[] args) {
+		ConfigurableApplicationContext context = new SpringApplicationBuilder(SampleRedisApplication.class)
+				.properties("spring.redis.lettuce.pool.enabled:false").run(args);
```

Comment:
There's no reason to put the property here. Why not put it in `application.properties` as usual?